intel-sgx/dcap-artifact-retrieval/src/provisioning_client/intel.rs

```diff
@@ -430,7 +430,7 @@ impl<'inp> ProvisioningServiceApi<'inp> for QeIdApi {
     fn build_request(&self, input: &Self::Input) -> Result<(String, Vec<(String, String)>), Error> {
         let api_version = input.api_version as u8;
         let url = format!(
-            "{}/sgx/certification/v{}/qe/identity",
+            "{}/sgx/certification/v{}/qe/identity?update=early",
```

Comment:
I think these need to be added in PCCS as well?

Reply:
Good catch! I've updated the last commit.

Reply:
Does this apply to Azure as well?

Reply:
The Azure client fetches the TcbInfo and QeIdentity data from Intel and thus also pull in the changes in this PR.